app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/AbilityScore.java

```diff
@@ -0,0 +1,55 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+class AbilityScore {
+    Integer value;
+    Integer modifier;
+    Integer tempValue;
+    Integer tempModifier;
+    Boolean useTemp;
+
+    public AbilityScore() {
```

Comment:
I think we should delete default constructor in future.

Reply:
Oh you're NUT!